### ISSUE TITLE
python example does not work any more

### ISSUE THREAD

**Issue by wenwenchenbosch** [GENIVI/iot-event-analytics#103]
I got the following warning, when I run the [python basic example](https://github.com/GENIVI/iot-event-analytics/tree/develop/src/sdk/python/examples/basic)

```
python run.py
2021-06-18T14:55:54.019Z INFO [ 61872ecc-79fe-4471-a764-1a010ad8a150.MqttClient-8bc83b20 ]: *****INFO***** Using topic namespace iotea/
2021-06-18T14:55:54.019Z INFO [ Talent.python-basic-talent-4497ba31 ]: Talent python-basic-talent-4497ba31 started successfully
2021-06-18T14:55:54.019Z INFO [ 61872ecc-79fe-4471-a764-1a010ad8a150.MqttClient-8bc83b20 ]: Connecting to mqtt://localhost:1883 ...
2021-06-18T14:55:54.023Z WARNING [ hbmqtt.mqtt.protocol.handler ]: Unhandled exception: 'Lock' object is not iterable
Connection failed: TypeError("'Lock' object is not iterable")
2021-06-18T14:55:55.025Z WARNING [ hbmqtt.mqtt.protocol.handler ]: Unhandled exception: 'Lock' object is not iterable
Reconnection attempt failed: TypeError("'Lock' object is not iterable")
2021-06-18T14:55:57.028Z WARNING [ hbmqtt.mqtt.protocol.handler ]: Unhandled exception: 'Lock' object is not iterable
Reconnection attempt failed: TypeError("'Lock' object is not iterable")
2021-06-18T14:55:59.030Z WARNING [ hbmqtt.mqtt.protocol.handler ]: Unhandled exception: 'Lock' object is not iterable
Reconnection attempt failed: TypeError("'Lock' object is not iterable")
2021-06-18T14:56:01.033Z WARNING [ hbmqtt.mqtt.protocol.handler ]: Unhandled exception: 'Lock' object is not iterable
```
After I send events using mqtt tool `node cli.js pub -c "mqtt://localhost:1883" -t "iotea/ingestion/events" -f "../../sdk/python/examples/basic/events.txt" --times 1 --delayMs 2000 --transform iotea.ts-now.jna`
no event reached the basic example.

I am using python 3.9.5.

**Comment by llakie** [comment]
@mariaivanova-git Can you check, if it has anything to do with the protocol gateway implementation

**Comment by mariaivanova-git** [comment]
The error seems to be due to the incompatibility of hbmqtt with python3.9 (https://github.com/beerfactory/hbmqtt/issues/223)
and has been recorded before (https://github.com/GENIVI/iot-event-analytics/issues/47).

Unfortunately hbmqtt does not provide a fix, so for the moment using a python version 3.7 or 3.8 could be the solution.

**Comment by llakie** [comment]
Fixed by PR #113
